content/applications/hr/employees/offboarding.rst

```diff
@@ -0,0 +1,77 @@
+===========
+Offboarding
+===========
+
+When an employee leaves the company, it is important to ensure their employee record is updated to
+reflect their departure, log the reason why, close any open activities associated with the
+employee, and provide them with any important documents.
+
+Archive an employee
+===================
+
+In Odoo, when an employee leaves the company they must be *archived*. To archive an employee, first
+navigate to the main **Employees** app. From here, locate the employee who is leaving the company,
```

Comment:
```suggestion
navigate to the :menuselection:`Employees app`. From here, locate the employee who is leaving the company,
```
when instructing the reader to "navigate" somewhere, the menuselection formatting is used 👍